pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -121,3 +130,28 @@ func (ci *cloudinfo) getNetwork(networkName string) (*networks.Network, error) {
 
 	return network, nil
 }
+
+func (ci *CloudInfo) getZones() ([]string, error) {
+	zones := []string{}
+	allPages, err := availabilityzones.List(ci.clients.computeClient).AllPages()
+	if err != nil {
+		return []string{}, (err)
```

Comment:
uhm... what does `(err)` mean?

Reply:
oh that's weird haha. I'm not sure where the parenthesis came from